packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -865,59 +903,60 @@ export class GarbageCollector implements IGarbageCollector {
     }
 
     /**
-     * Validates that all new references are correctly identified and processed. The basic principle for validation is
-     * that we should not have new references in the reference graph (GC data) that have not been notified to the
-     * garbage collector via `referenceAdded`.
-     * We validate that the references in the current reference graph should be a subset of the references in the last
-     * run's reference graph + references since the last run.
+     * Finds outbound references to datastores in the current GC graph that aren't in the previous GC graph and returns
+     * them if they are not added by GC's `addedOutboundReference`.
+     *
+     * In more simple terms:
+     * Missing Explicit References = Current References - Previous References - Explicitly Added References;
+     *
+     * Currently outbound references are references from a parent node to child datastore node
+     *
      * @param currentGCData - The GC data (reference graph) from the current GC run.
+     * @param previousGCData - The GC data (reference graph) from the previous GC run.
+     * @param explicitReferences - New references added by GC explicity between the previous and the current run.
```

Comment:
remove "by GC" here. The references are not added by GC. GC is only notified of these.